sstables_loader.cc

```diff
@@ -81,10 +84,12 @@ class send_meta_data {
         llog.trace("send_meta_data: send mf to node={}, size={}", _node, size);
         co_return co_await _sink(fmf, streaming::stream_mutation_fragments_cmd::mutation_fragment_data);
     }
-    future<> finish(bool failed) {
+    future<> finish(bool failed, bool aborted = false) {
```

Comment:
Best to avoid default arguments, they lead to mistakes.

Reply:
100% with you but for some reason this approach is not too popular

Reply:
What do you mean? Did someone else tell you to use the default arg?

We do use defaults when a change to a signature would be very painful and the new arg is used by a small minority of callers. But here, I don't think there are a lot of callers overall, so we can just remove the default and adjust all callers.